### PR TITLE
update a broken link

### PR DESCRIPTION
**Pull request by OnkarRuikar**

### CODE REVIEW

File: files/en-us/web/api/webxr_device_api/inputs/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ WebXR supports a variety of different types of devices to handle targeting and a
 - Motion-sensing controllers, which use accelerometers, magnetometers, and other sensors for motion tracking and targeting and may additionally include any number of buttons, joysticks, thumbpads, touchpads, force sensors, and so on to provide additional input sources for both targeting and selection.
 - Squeezable triggers or glove grip pads to provide squeeze actions.
 - Voice commands using speech recognition.
-- Spatially-tracked articulated hands, such as {{interwiki("wikipedia", "haptic gloves")}} can provide both targeting and squeeze actions, as well as selection if outfitted with buttons or other sources of selection actions.
+- Spatially-tracked articulated hands, such as [Wired glove](https://en.wikipedia.org/wiki/Wired_glove) can provide both targeting and squeeze actions, as well as selection if outfitted with buttons or other sources of selection actions.
```
Comment (teoli2003) [inline files/en-us/web/api/webxr_device_api/inputs/index.md:44]:
```suggestion
- Spatially-tracked articulated hands, such as [wired gloves](https://en.wikipedia.org/wiki/Wired_glove) can provide both targeting and squeeze actions, as well as selection if outfitted with buttons or other sources of selection actions.
```

**Review by teoli2003 — APPROVED** [review]
👍 

Note that I would like to remove all usage of `{{interwiki}}` on mdn/content (but not yet on mdn/translated-content).

**Comment by OnkarRuikar** [comment]
> Note that I would like to remove all usage of `{{interwiki}}` on mdn/content

🙋🏻‍♂️ I'll do it. 
There are around 576 of them :smile:

**Comment by teoli2003** [comment]
Cool. They may lead to "hidden" broken links, though.

**Comment by OnkarRuikar** [comment]
> They may lead to "hidden" broken links, though.

hidden broken? could you elaborate?

***

I'll automate it. 
The same macro implementation will be used to figure out replacements. And will make a simple `fetch` request to make sure the replacements return http 200 code.

Should I use English links `https://en.wikipedia.org/wiki/Wired_glove` or generic ones `https://wikipedia.org/wiki/Wired_glove` (I think these redirect to localized ones)?

**Comment by teoli2003** [comment]
The `fetch()` request will "unhide" them.

English please, in some languages the link doesn't match.

**Pull request merged**